creatorkit/Toolbox.md

```diff
@@ -0,0 +1,157 @@
+
+<code>$ sudo miniterm.py /dev/ttyUSB0 -b 115200</code>
+
+To determine the address to use (in this case /dev/ttyUSB0) you can run
+
+<code>$ ls /dev/ttyU*</code>
+
+Once connected, you should have access to a terminal.
+
+## Creating a Bootable USB Drive/microSD Card
+
+Note:
+
+* You cannot boot from USB Drive while also powering the board via microUSB as they both use the same connector (the one next to the power socket). You will need to power the board using a power supply instead, or use a microSD card as your boot device.
+* To connect a USB pen drive, you need a USB A to microUSB OTG cable.
+
+### Formatting your USB Drive/microSD Card
+
+Before you connect your USB Drive or microSD Card (called "Storage Device" from this point on) to your Ubuntu machine, run the following:
+
+<code>$ df</code>
+
+This will give a list of storage devices that are currently connected. Now connect your storage device and run the same command again. You should now see an additional line that will look something like the following:
+
+<code>/dev/sdb1        7823232    182204   7641028   3% /media/home/name_of_storage_device</code>
+
+In this example /dev/sdb1 is the part we need in future commands - You will need to substitute your path for /dev/sdb1 if it is different. The following commands will format the storage device - make sure you have stored any important data elsewhere. Also be 100% sure of the path of the storage device before running these commands.
+
+<code>$ umount /dev/sdb1</code>
+
+<code>$ sudo mkfs.ext4 /dev/sdb1</code>
+
+Remount the storage device (the simplest way is to remove it and reconnect it so it mounts automatically).
+
+You can now take the Ci40 Creator Kit release from Github and copy it to the storage device. You can extract the .tar file directly to the storage device using a command similar to the following (but using your paths).
+
+<code>$ sudo tar -zxvf ~/downloads/rootfs.tar -C /media/home/name_of_storage_device</code>
+
+<code>$ sync</code>
+
+### Booting from USB Drive/SD Card
+
+In order to boot from USB you need to connect it via an On-the-go cable to the microUSB port next to the power connector. Once that is connected reboot the board. You will then have a window of opportunity to interrupt the default boot process (booting from the flash memory) by pressing a key in the serial terminal. This will give you access to the bootloader terminal.
+
+To boot from USB:
+
+<code># run usbboot</code>
+
+To boot from microSD:
+
+<code># run mmcboot</code>
+
+## Connecting Ci40 to the Internet
+
+Ci40 supports Ethernet for network connectivity.
+
+## Programming a 6LoWPAN Clicker
+
+Before programming a Clicker board you need to apply power. You can use batteries, USB or (for purely for programming purposes) a Pickit programmer.
```

Comment:
PicKit3 programmer to be specific, so that people don't go and buy any other PicKit programmer